en_us.lang

```diff
@@ -157,200 +157,200 @@ gui.xaero_transfer_all=Transfer All Waypoints
-gui.xaero_box_entity_nametags=Display entity name under the dot for entities that have a name tag.
-gui.xaero_box_teleport_default_command=The command used for waypoints teleportation if a server-specific command is not set in the waypoints menu Options screen.
-gui.xaero_waypoints_edit_mode_individually=One-by-one
-gui.xaero_waypoints_edit_mode_all=All
+gui.xaero_show_angles=Display Facing Angles
+gui.xaero_box_entity_nametags=Display entity name under the dot on the minimap for entities that have name tags.
+gui.xaero_box_teleport_default_command=The command used for waypoint teleportation if a world-specific or a server-specific command is not set in the Options menu on the waypoint screen.
+gui.xaero_waypoints_edit_mode_individually=One by One
+gui.xaero_waypoints_edit_mode_all=All at Once
 gui.xaero_compass=Compass
-gui.xaero_cave_maps_depth=Cave Maps Depth
+gui.xaero_cave_maps_depth=Cave Mode Depth
 gui.xaero_waypoints_edit_reset=Reset
 gui.xaero_main_entity_as=Main Entity As
 gui.xaero_arrow=Arrow
 gui.xaero_dot=Dot
 gui.xaero_crosshair=Crosshair
-gui.xaero_teleport_coordinates_hidden=Unable to teleport because it would tell you the waypoint coordinates. Disable "Hide Waypoint Coords" to be able to freely teleport again.
-gui.xaero_hide_wp_coords=Hide Waypoint Coords
+gui.xaero_teleport_coordinates_hidden=Unable to teleport, because it could expose the waypoint coordinates. Disable the "Hide Waypoint Coordinates" option to be able to freely teleport again.
+gui.xaero_hide_wp_coords=Hide Waypoint Coordinates
 gui.xaero_waypoints_edit_show=Show
 gui.xaero_waypoints_edit_hide=Hide
 gui.xaero_teleport_anyway=Teleport Anyway
-gui.xaero_teleport_shows_coordinates=Shows you the coordinates!
+gui.xaero_teleport_shows_coordinates=It will expose the coordinates!
 gui.xaero_render_all_wp_sets=Render All Waypoint Sets
 gui.xaero_waypoints_global=Global
 gui.xaero_waypoints_local=Local
 gui.xaero_player_arrow_opacity=Arrow Opacity
 gui.xaero_box_waypoints_distance=Maximum render distance for local waypoints. Global waypoints are not affected.
 gui.xaero_connect_with_auto=Add Sub-World Connection
 gui.xaero_disconnect_from_auto=Delete Sub-World Connection
-gui.xaero_world_connection_tooltip=Connect/disconnect selected sub-world to/from the current "auto" one. Only connect sub-worlds that are from the same world save. For example, if a vanilla Nether portal takes you from sub-world A to sub-world B, then you can connect them. Sub-worlds connected to your current one are always at the top of the list and are marked by a * symbol.
+gui.xaero_world_connection_tooltip=Connect/disconnect selected sub-world to/from the current automatic one. Connect only sub-worlds that are from the same world save. For example, if a vanilla Nether portal takes you from sub-world A to sub-world B, then you can safely connect them. Other sub-worlds connected to the current one are always at the top of the list and are marked by an asterisk (symbol: *).
```

Comment:
I think "auto" is better because that's the word displayed in the dropdown menu.

Reply:
OK, I'm going to change "automatic" to "auto" here and in all other similar places (`gui.xaero_make_automatic_msg2`, `gui.xaero_make_multi_automatic_msg2`, and `gui.xaero_teleport_not_connected`) as well, but I'm not convinced that this is a positive change, to be honest.

Reply:
It's definitely a positive change imo

Reply:
Oh, the word auto was intentionally in quotes before, which was a lot better imo.

Reply:
I thought that since `gui.xaero_make_automatic` and `gui.xaero_make_multi_automatic` don't have "auto" with quotes, this line shouldn't either. Could you please tell me why they are needed?

Reply:
Missed this one too. They are needed because it's a tooltip, which is supposed to make things clear.

Reply:
I like the idea with parentheses, but I still don't understand why you would like to have quotation marks here. The only use case that I think can apply here is to use them as scare quotes, but "auto" here really means auto (without any hidden meaning).

Reply:
Idk, it makes it look like we're talking about a specific thing.

Reply:
I'm not sure either. Maybe ask someone else? :/